### PR TITLE
Update the timeout of the clean-up cluster step to 1h

### PR DESCRIPTION
**Pull request by ytimocin**
# Description
Update the timeout of the clean-up cluster step to 1h.

## Type of change
- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

### CODE REVIEW

**Review by vishwahiremat — APPROVED** [review]

File: .github/workflows/long-running-azure.yaml [hunk 1]
```diff
@@ -512,7 +514,7 @@ jobs:
       - name: Clean up cluster
```
Comment (sk593) [inline .github/workflows/long-running-azure.yaml:514]: Do we need to have two cleanup cluster steps? These both seem to be part of the test job

Comment (ytimocin) [reply .github/workflows/long-running-azure.yaml:514]: One cleans up before the tests and the other one after the tests. This is to be extra defensive. We want to have a clean start before running the tests and want to make sure we delete everything after we run the tests.

**Review by sk593 — APPROVED** [review]

File: .github/workflows/long-running-azure.yaml (hunk 1 above)
Comment (sk593) [reply .github/workflows/long-running-azure.yaml:514]: makes sense. PR lgtm then

**Comment by lakshmimsft** [comment]
@ytimocin  I am curious if this will alleviate the issue. If we are terminating the script in an hour and there are still resources/finalizers than need clean up, will they be cleaned up while the finalizers are still attached during the second run on the script? Based on the overlapping runs for multiple days, it seems like multiple runs of the script did not help clean it up.
let's try this. If it works, great, if not, maybe after the first run of the script, we can cycle through all resources in the namespace with 'Terminating' state and log them, so we narrow down what the finalizers are waiting on.

**Review by lakshmimsft — APPROVED** [review]

**Comment by ytimocin** [comment]
> @ytimocin I am curious if this will alleviate the issue. If we are terminating the script in an hour and there are still resources/finalizers than need clean up, will they be cleaned up while the finalizers are still attached during the second run on the script? Based on the overlapping runs for multiple days, it seems like multiple runs of the script did not help clean it up. Atleast we will not wait for the script to complete over an hour, so that is good. let's try this. If it works, great, if not, maybe after the first run of the script, we can cycle through all resources in the namespace with 'Terminating' state and log them, so we narrow down what the finalizers are waiting on.

The problem with 6-hour timeout is that it doesn't do clean-up during that time. It just waits for the finalizers to finalize. But that doesn't happen. If the workflow waited for 6 days, it wouldn't also happen. The problem is something else.

I wanted to reduce the time to 1h so that we don't wait 6 hours for a run to be finalized.

Thanks for the feedback and the approval @lakshmimsft!

**Pull request merged**